src/dashboard/src/components/dashboard/Dashboard.tsx

```diff
@@ -173,6 +173,26 @@ export const Dashboard = () => {
     [download],
   );
 
+  const renderStorageTrendsChart = (isLarge: boolean) => {
```

Comment:
Should this use a `React.useCallback()`?

Reply:
for the callback, should I use all the variables this component depends on as dependencies?  
ex: ```[
      dashboardServerItem,
      dashboardServerItems,
      dashboardTenant,
      dashboardOrganization,
      dashboardOperatingSystemItem,
      handleExport,
]```

Reply:
If you test using them all you'll find out if it gets in an infinite loop.  But generally you want all of them.